plenum/test/checkpoints/test_backup_replica_resumes_ordering_on_lag_in_checkpoints.py

```diff
@@ -67,11 +79,11 @@ def test_backup_replica_resumes_ordering_on_lag_in_checkpoints(
 
     # Ensure that the replica has not ordered any batches
     # after the very first one
-    assert slow_replica.last_ordered_3pc == (view_no, 2)
+    assert slow_replica.last_ordered_3pc == (view_no, batches_count)
 
     # Ensure that the watermarks have not been shifted since the view start
-    assert slow_replica.h == 0
-    assert slow_replica.H == LOG_SIZE
+    assert slow_replica.h == low_watermark
+    # assert slow_replica.H == sys.maxsize
```

Comment:
Why did we remove `slow_replica.H` check? Should it be `assert slow_replica.H == low_watermark + LOG_SIZE`?